package.json

```diff
@@ -81,26 +88,25 @@
     "sass-functions": "^1.4.0",
     "serve-static": "^1.13.2",
     "sinon": "^9.2.1",
-    "string-validate": "^3.4.0",
-    "stylelint": "^13.7.2",
+    "string-validate": "^3.5.0",
+    "stylelint": "^13.8.0",
     "stylelint-config-nebenan": "^1.6.2",
-    "terser": "^5.3.8",
-    "tiny-lr": "^1.1.1",
+    "tiny-lr": "^2.0.0",
     "vinyl-source-stream": "^2.0.0"
   },
   "dependencies": {
     "nebenan-react-datepicker": "^1.1.0"
   },
   "peerDependencies": {
```

Comment:
updating peer deps is a breaking change -> major version bump

Reply:
all of these are minor bumps, it should raise any errors only force npm to decache and fetch latest

Reply:
I think it will show an peer dep error because the version statements inside deps/devDeps don't cover the min-version of the peerDep

devDeps:
- clsx: ^1.1.0

peerDep of other package:
- clsx: ^1.1.1

Didn't test it though, thought I saw it somewhere some days ago.

Reply:
`npm run reinstall` solves it worst case. will check though